api/matlab/arrus/BModeDisplay.m

```diff
@@ -3,38 +3,124 @@
     %
     % Currently is implemented as a simple MATLAB figure with dynamically
     % updated content.
-    %
+    % 
+    % :param reconstructionObject: object of class "Reconstruction" \
+    %   (obligatory input unless old syntax is used)
+    % :param dynamicRange: two-element vector [min, max], dynamic range \
+    %   limits to apply (optional name-value argument)
+    % :param cineLoopLength: positive scalar, number of frames stored in \
+    %   cineloop (optional name-value argument)
+    % :param persistence: persistence filtration weights (vector) or \
+    %   length (scalar) (optional name-value argument)
+    % :param bmodeTgc: applies linear TGC equal to bmodeTgc[dB]/depth \
+    %   (scalar) (optional name-value argument)
+    % :param bmodeAutoTgcResp: applies linear TGC that adapts in time to \
+    %   the imaging conditions, defines the responsiveness of the \
+    %   adaptation, <0,1> (scalar) (optional name-value argument)
```

Comment:
The 'responsiveness of the adaptation' could be explained more clear.

Reply:
"(0-no adaptation, 1-instant adaptation)" added